modules/catalog/README.md

```diff
@@ -1,7 +1,54 @@
 # Catalog module
 
-This module provides database catalog service implementation and descriptor for catalog objects.
+Catalog is a component that responsible for managing descriptors of objects available in cluster,
+as well as serving as single source of truth for other components to acquire an actual state of
+the schema.
```

Comment:
should be explained which of schema?

Reply:
this one looks like artifact of uncompleted change integration... Catalog manages different schemas,  so it's kinda unclear what schema I was talking about :( anyway, rephrase this once again, hope now it's better